sdk/docs/core/resources/mqtt_connection.puml

```diff
@@ -15,6 +15,8 @@ state ConnectionManagement {
 
     state Started {
         state Connecting
+        Connecting : <b>entry</b>/ stopwatch_reset()
+        Connecting : <b>exit</b>/ stopwatch_gettime()
```

Comment:
A bit confused about these two stopwatch functions - does reset start the stopwatch or clear it? And does get time just save the value or stop the stopwatch as well?
```suggestion
        Connecting : <b>exit</b>/ stopwatch_get_time()
```

Reply:
There is no stopwatch technically, what happens is that reset records the current system time, gettime will record that again at a later point and calculate how much time has elapsed